flink-cdc-cli/src/test/java/com/ververica/cdc/cli/CliFrontendTest.java

```diff
@@ -83,16 +83,6 @@ void testGlobalPipelineConfigParsing() throws Exception {
         assertThat(executor.getGlobalPipelineConfig().toMap().get("foo")).isEqualTo("bar");
     }
 
-    @Test
-    void testAdditionalJar() throws Exception {
-        String aJar = "/foo/jar/a.jar";
-        String bJar = "/foo/jar/b.jar";
-        CliExecutor executor =
-                createExecutor(
-                        pipelineDef(), "--flink-home", flinkHome(), "--jar", aJar, "--jar", bJar);
-        assertThat(executor.getAdditionalJars()).contains(Paths.get(aJar), Paths.get(bJar));
```

Comment:
This test is use to verify the correctness of `--jar` option, it's better to remain it.